vnext/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -186,6 +196,12 @@ void ScrollViewShadowNode::updateProperties(const folly::dynamic &&reactDiffMap)
       if (valid) {
         ScrollViewUWPImplementation(scrollViewer).SnapToEnd(snapToEnd);
       }
+    } else if (propertyName == "keyboardDismissMode") {
+      m_dismissKeyboardOnDrag = false;
+      if (propertyValue.isString()) {
+        m_dismissKeyboardOnDrag = (propertyValue.getString() == "on-drag");
+        RegisterSIPEventsWhenNeeded();
```

Comment:
>RegisterSIPEventsWhenNeeded [](start = 8, length = 27)

Do you need to unRegisterSIPEventsWhenNeeded in else? otherwise there is a callback is never be released.

Reply:
It's fine, we tear down event registration in the destructor of keyboardEventHandler, and we release keyboardEventHandler in ~ScrollViewShadowNode.

---
In reply to: [348693046](https://github.com/microsoft/react-native-windows/pull/3665#discussion_r348693046) [](ancestors = 348693046)